dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/PatientController.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// MIT License
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the ""Software""), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Mvc;
+using System.Threading.Tasks;
+using System.Web.Configuration;
+using HealthVaultProviderManagementPortal.Models.Patient;
+using Microsoft.HealthVault.Configuration;
+using Microsoft.HealthVault.Web.Attributes;
+using Microsoft.Rest;
+using Microsoft.Rest.Serialization;
+using Newtonsoft.Json;
+using NodaTime;
+using NodaTime.Serialization.JsonNet;
+using static HealthVaultProviderManagementPortal.Helpers.RestClientFactory;
+
+namespace HealthVaultProviderManagementPortal.Controllers
+{
+    /// <summary>
+    /// Controller for creating and sending invitations to remote patients
+    /// </summary>
+    [RequireSignIn]
+    public class PatientController : Controller
+    {
+        /// <summary>
+        /// Gets a the timeline for a patient
+        /// </summary>
+        public async Task<ActionResult> Index(Guid personId, Guid recordId, DateTime? startDate, DateTime? endDate)
+        {
+            if (startDate == null)
```

Comment:
It is. I wasn't sure if I wanted to use post for the filters. I thought it made sense for them to show in the URL. Thoughts?